lib/ecto/query/planner.ex

```diff
@@ -1285,7 +1346,7 @@ defmodule Ecto.Query.Planner do
 
   ## Helpers
 
-  @exprs [distinct: :distinct, select: :select, from: :from, join: :joins,
+  @exprs [with_ctes: :with_ctes, distinct: :distinct, select: :select, from: :from, join: :joins,
```

Comment:
```suggestion
  @exprs [with_cte: :with_ctes, distinct: :distinct, select: :select, from: :from, join: :joins,
```

IIRC the left side is used for error reporting, so it needs to be the name of the macro/operation the user calls.